### PR TITLE
Updating docs

### PR DESCRIPTION
**Pull request by lode**
While going through the docs to get this lib up and running, I've noticed these improvements.

Hope it helps.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
+        "php": ">=5.5",
```
Comment (Ph3nol) [inline composer.json]: Hum... Any reason about this very personal choice?

Comment (lode) [inline composer.json]: Not a personal choice, as noted in the commit message, the zend libs require this for me. You don't have that issue?

Comment (Ph3nol) [inline composer.json]: Oh sorry! My bad! Didn't see it, seems to be real... :/ https://travis-ci.org/Ph3nol/NotificationPusher/builds/77487072

Thanks for this!

Comment (lode) [inline composer.json]: It is weird though that travis doesn't fail installing dependencies for the master branch...

**Comment by Ph3nol** [comment]
Thank you for this! :wink:

**Pull request merged**